spec/reek/code_comment_spec.rb

```diff
@@ -1,103 +1,129 @@
-      expect(described_class.new("# fred\n#  ")).not_to be_descriptive
+      comment = FactoryGirl.build(:code_comment, comment: "# alpha\n#  ")
+      expect(comment).not_to be_descriptive
     end
+
     it 'accepts a 2-word comment' do
-      expect(described_class.new('# fred here  ')).to be_descriptive
+      comment = FactoryGirl.build(:code_comment, comment: '# alpha bravo  ')
+      expect(comment).to be_descriptive
     end
+
     it 'accepts a multi-word comment' do
-      comment = "# fred here \n# with \n   # biscuits "
-      expect(described_class.new(comment)).to be_descriptive
+      comment = FactoryGirl.build(:code_comment, comment: "# alpha bravo \n# charlie \n   # delta ")
+      expect(comment).to be_descriptive
     end
   end
 
-  context 'comment config' do
+  context 'good comment config' do
     it 'parses hashed options' do
-      comment = '# :reek:Duplication { enabled: false }'
-      config = described_class.new(comment).config
-      expect(config).to include('Duplication')
-      expect(config['Duplication']).to include('enabled')
-      expect(config['Duplication']['enabled']).to be_falsey
+      comment = FactoryGirl.build(:code_comment, comment: '# :reek:DuplicateMethodCall { enabled: false }')
+
+      expect(comment.config).to include('DuplicateMethodCall')
+      expect(comment.config['DuplicateMethodCall']).to include('enabled')
+      expect(comment.config['DuplicateMethodCall']['enabled']).to be_falsey
     end
 
     it "supports hashed options with the legacy separator ':' after the smell detector" do
-      comment = '# :reek:Duplication: { enabled: false }'
-      config = described_class.new(comment).config
-      expect(config).to include('Duplication')
-      expect(config['Duplication']).to include('enabled')
-      expect(config['Duplication']['enabled']).to be_falsey
+      comment = FactoryGirl.build(:code_comment, comment: '# :reek:DuplicateMethodCall: { enabled: false }')
+
+      expect(comment.config).to include('DuplicateMethodCall')
+      expect(comment.config['DuplicateMethodCall']).to include('enabled')
+      expect(comment.config['DuplicateMethodCall']['enabled']).to be_falsey
     end
 
     it 'parses multiple hashed options' do
-      config = described_class.new('
-        # :reek:Duplication { enabled: false }
+      original_comment = <<-EOF
+        # :reek:DuplicateMethodCall { enabled: false }
         # :reek:NestedIterators { enabled: true }
-      ').config
-      expect(config).to include('Duplication', 'NestedIterators')
-      expect(config['Duplication']).to include('enabled')
-      expect(config['Duplication']['enabled']).to be_falsey
-      expect(config['NestedIterators']).to include('enabled')
-      expect(config['NestedIterators']['enabled']).to be_truthy
+      EOF
+      comment = FactoryGirl.build(:code_comment, comment: original_comment)
+
+      expect(comment.config).to include('DuplicateMethodCall', 'NestedIterators')
```

Comment:
I'd prefer to avoid the duplicate `comment.config` here and in the other specs in this file by assigning to a local variable (as it was before).